openssl-sys/build.rs

```diff
@@ -44,7 +44,11 @@ fn main() {
     let libs = if (version.contains("0x10001") ||
                    version.contains("0x10002")) &&
                   target.contains("windows") {
-        ["ssleay32", "libeay32"]
+        if target.contains("windows-gnu") {
+            ["ssl", "crypto"]
```

Comment:
@sfackler I downloaded a copy of OpenSSL for Windows from https://slproweb.com/products/Win32OpenSSL.html. They were using `libeay32.dll` & `ssleay32.dll` for 1.0.* release but now using `libcrypto-1_1-x64.dll` & `libssl-1_1-x64.dll` for 1.1.0.